crane/views/v2.py

```diff
@@ -77,7 +78,8 @@ def name_redirect(relative_path):
         if schema2_data or manifest_list_data:
             # if it is a newer docker client it sets accept headers to manifest schema 1, 2 and list
             # if it is an older docker client, he doesnot set any of accept headers
-            accept_headers = request.headers.get('Accept')
+            accept_headers = get_accept_headers(request)
+            log.debug("Accept: %s", accept_headers)
```

Comment:
please remove the log statement, this information will not help anyone without a deep understanding of docker engine insides

Reply:
I don't anything about docker/crane, but I think someone that someone who turns on debug logging would have that deep understanding and want logging like this.

Reply:
I agree with bmbouter. More than once I wanted to have more information, and I would prefer to do that without changing code (just changing config). That being said, I shouldn't log.debug twice.

Reply:
it is not about docker/crane but about docker-engine, the client for pulling the docker content and docker api. I still kinda think this is not very helpful, but if we want to keep this then please at least update to a more meaningful message something like " Accept headers sent from docker client" or "Accept header sent during docker pull"